src/Components/Profile.js

```diff
@@ -40,6 +45,7 @@ Profile.propTypes = {
   username: PropTypes.string.isRequired,
   profile: PropTypes.shape({
     name: PropTypes.string.isRequired,
+    type: PropTypes.string.isRequired,
```

Comment:
I think it is required to use a type to add a condition for differentiating between profiles.
- Communities always going to use profile type.
- If the type is empty in data, then still it is not going to affect in any way,

Any other way to achieve something?

Reply:
Just reading up a little on propTypes...

The isRequired should be removed because it is an optional prop and is not always guaranteed to be there. We don't want it throwing an error when it doesn't get the prop from personal profiles.

Then where you check for profile type you add an extra check like this ```profile.type && profile.type === "community``` to check first that profile.type exists

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Thanks for the help, Emma.
I forgot to check the error, will keep it in mind.

I have made changes as per the suggestions:
- I have removed the isRequied from type.
- `{profile.type && profile.type === 'community' &&`